reactive/kubernetes_master.py

```diff
@@ -2402,3 +2408,18 @@ def get_dns_provider():
 
     leader_set(auto_dns_provider=dns_provider)
     return dns_provider
+
+
+@when('kube-control.connected')
+@when_not('kubernetes-master.sent-registry')
+def send_registry_location():
+    registry_location = hookenv.config('image-registry')
+    kube_control = endpoint_from_flag('kube-control.connected')
+    kube_control.set_registry_location(registry_location)
+    set_flag('kubernetes-master.sent-registry')
+
+
+@when('kube-control.connected')
```

Comment:
Do we want to rely on kube-control being connected here? I can contrive a situation where this isn't true and you change the config and then it isn't properly reflected. It's reaching, but also an easy change.